### DISCUSSION TITLE
Mutation C#

### DISCUSSION THREAD

**Discussion by Timit0** [nathanhoad/godot_dialogue_manager#453] [Q&A]
Hi how can I call a c# property in a dialogue ?

![Capture d’écran 2024-01-23 085654](https://github.com/nathanhoad/godot_dialogue_manager/assets/132546200/ec73bb54-1f73-470b-a14e-947880dc0638)

![Capture d’écran 2024-01-23 085700](https://github.com/nathanhoad/godot_dialogue_manager/assets/132546200/5c5f68c0-4594-4f37-a331-70c25f5d58f2)

**Comment by DanielSparta** [comment]
Tell me what your trying to do and I will try to help you

**Reply by Timit0** [reply]
I want to call "PlayerName" in my dialogue

**Comment by nathanhoad** [comment]
Is `Tt` set up to be an autoload?
